src/components/atoms/Input/Component.tsx

```diff
@@ -21,104 +21,95 @@ const renderText = (content: any, defaultProps: any, style: StyleProp<any>) =>
 export type InputProps = React.ComponentPropsWithRef<typeof TextInput> & {
   containerStyle?: StyleProp<ViewStyle>;
   disabled?: boolean;
-  disabledInputStyle?: StyleProp<TextStyle>;
-  focusInputColor?: string;
   inputContainerStyle?: StyleProp<ViewStyle>;
   inputStyle?: StyleProp<TextStyle>;
-  InputComponent?: typeof React.Component;
-  errorProps?: any;
   errorStyle?: StyleProp<TextStyle>;
   errorMessage?: string;
   label?: string | React.ReactNode;
   labelStyle?: StyleProp<TextStyle>;
-  labelProps?: any;
-  renderErrorMessage?: boolean;
   showClearText?: boolean;
   iconStyle?: StyleProp<TextStyle>;
   multiline?: boolean;
   numberOfLines?: number;
   maxLength?: number;
+  onClear?: () => void;
 };
 
-const Input: FunctionComponent<InputProps> = ({
-  containerStyle,
-  disabled,
-  disabledInputStyle,
-  inputContainerStyle,
-  InputComponent = TextInput,
-  inputStyle,
-  errorProps,
-  errorStyle,
-  errorMessage,
-  label,
-  labelStyle,
-  labelProps,
-  renderErrorMessage = true,
-  style,
-  showClearText,
-  iconStyle,
-  multiline,
-  numberOfLines,
-  maxLength,
-  ...attributes
-}) => {
-  const hideErrorMessage = !renderErrorMessage && !errorMessage;
+const Input: FunctionComponent<InputProps> = forwardRef<any, InputProps>(
+  (
+    {
+      containerStyle,
+      disabled,
+      inputContainerStyle,
+      inputStyle,
+      errorStyle,
+      errorMessage,
+      label,
```

Comment:
please use props name: "style" as much as possible

Reply:
@nhat-ndm1193 can it text or title which defined in our BE ?

Reply:
@zrg-team yah lemme check